### PR TITLE
Exercises Woche2

### PR DESCRIPTION
**Pull request by CelinaRiegel**

### CODE REVIEW

File: Exercises/Celina/exercises_02.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                first_pos= commands[i+1]
+                second_pos=commands[i+2]
+                thierd_pos=commands[i+3]
+                sum=commands[first_pos] + commands[second_pos]
+                commands[thierd_pos]=sum
+            case 2:
+                first_pos= commands[i+1]
+                second_pos=commands[i+2]
+                thierd_pos=commands[i+3]
+                sum=commands[first_pos] * commands[second_pos]
```
Comment (LeMixer) [inline Exercises/Celina/exercises_02.py:47]: Misleading variable `sum` when multiplying numbers the result is not a sum, but rather a product.

File: Exercises/Celina/exercises_02.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+commands = [1, 12, 2, 3, 1, 1, 2, 3, 1, 3, 4, 3, 1, 5, 0, 3, 2, 1, 9, 19, 1, 5, 19, 23, 1, 6, 23, 27, 1, 27, 10, 31, 1, 31, 5, 35, 2, 10, 35, 39, 1, 9, 39, 43, 1, 43, 5, 47, 1, 47, 6, 51, 2, 51, 6, 55, 1, 13, 55, 59, 2, 6, 59, 63, 1, 63, 5, 67, 2, 10, 67, 71, 1, 9, 71, 75, 1, 75, 13, 79, 1, 10, 79, 83, 2, 83, 13, 87, 1, 87, 6, 91, 1, 5, 91, 95, 2, 95, 9, 99, 1, 5, 99, 103, 1, 103, 6, 107, 2, 107, 13, 111, 1, 111, 10, 115, 2, 10, 115, 119, 1, 9, 119, 123, 1, 123, 9, 127, 1, 13, 127, 131, 2, 10, 131, 135, 1, 135, 5, 139, 1, 2, 139, 143, 1, 143, 5, 0, 99, 2, 0, 14, 0]
+
+def weird_shuffle(commands):
+    i=0
+    while i <= len(commands):
+        match commands[i]: 
+            case 1:
+                first_pos= commands[i+1]
+                second_pos=commands[i+2]
+                thierd_pos=commands[i+3]
```
Comment (LeMixer) [inline Exercises/Celina/exercises_02.py:40]: typo

File: Exercises/Celina/exercises_02.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+list_1=[1,3,"hallo",7,"0",7.8,0,3/4,678.9785,"abcde"]
+list_2=[nan,1/3,"abs","23a","1234",678.9]
+
+def split_list(list):
+    number_list=[]
+    not_number_list=[]
+    for i in list:
+        try:
+            float(i)
+            if i != i:
```
Comment (LeMixer) [inline Exercises/Celina/exercises_02.py:77]: This condition will never run since this is a contradiction.

File: Exercises/Celina/exercises_02.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                not_number_list.append(i)
+            else:
+                number_list.append(i)
+        except ValueError:
+            not_number_list.append(i)
+    return number_list, not_number_list
```
Comment (LeMixer) [inline Exercises/Celina/exercises_02.py:83]: The task is to sort values which can be interpreted as a number and 
>all strings which contain just one character

**Review by LeMixer — CHANGES_REQUESTED** [review]

File: Exercises/Celina/exercises_02.py (hunk 3 above)
Comment (CelinaRiegel) [reply Exercises/Celina/exercises_02.py:77]: That was my solution to put nan in the not_number list.

**Pull request merged**